br/pkg/aws/ebs.go

```diff
@@ -342,25 +347,7 @@ func (e *EC2Session) waitDataFSREnabled(snapShotIDs []*string, targetAZ string)
 		return errors.Errorf("specified snapshot [%s] is not found", *snapShotIDs[0])
 	}
 
-	for _, s := range resp.Snapshots {
-		if *s.VolumeSize > maxVolumeSize {
-			maxVolumeSize = *s.VolumeSize
-		}
-	}
-
-	// Calculate the time in minutes to fill 1.0 credit according to
-	// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-fast-snapshot-restore.html#volume-creation-credits
-	// 5 minutes more is just for safe
-	fillElapsedTime := 60.0/(min(10, 1024.0/(float64)(maxVolumeSize))) + 5
-
-	// We have to sleep for at least fillElapsedTime minutes in order to make credits are filled to 1.0
-	// Let's heartbeat every 5 minutes
-	for time.Since(start) <= time.Duration(fillElapsedTime)*time.Minute {
-		log.Info("FSR enablement is ongoing, going to sleep for 5 minutes...")
-		time.Sleep(5 * time.Minute)
-	}
-
-	// Wait that all snapshot has enough fsr credit balance, it's very likely true since we have wait for long enough
+	// Wait that all snapshot has enough fsr credit balance
```

Comment:
Previously, we will wait based on the estimated fsr enablement time. But if the restore failed, and restarted from scratch, we don't need to wait for such a long time.

Reply:
This change doesn't seem related to this PR at all. In general, I encourage you to make separate PRs for logically unrelated PRs.